distutils/cmd.py

```diff
@@ -306,7 +322,9 @@ def set_undefined_options(self, src_cmd, *option_pairs):
             if getattr(self, dst_option) is None:
                 setattr(self, dst_option, getattr(src_cmd_obj, src_option))
 
-    def get_finalized_command(self, command, create=True):
+    # NOTE: Because distutils is private setuptools implementation and we don't need to re-expose all commands here,
+    # we're not overloading each and every command possibility.
```

Comment:
```suggestion
    # NOTE: Because distutils is private to Setuptools and not all commands are exposed here,
    # not every possible command is enumerated in the signature.
```